CHANGELOG.md

```diff
@@ -62,6 +45,57 @@ Please also have a look at our
 @ziegenberg is a new contributor to this release and did a lot of the heavy
 lifting. Thanks! :heart:
 
+## 8.7.0: Add support for PHP 8.4
+
+### Added
+
+- Add support for PHP 8.4 (#675, #701, #746, #751)
+
+### Changed
+
+- Mark parsing-internal classes and methods as `@internal` (#711)
+- Block installations on unsupported higher PHP versions (#691)
+
+### Deprecated
+
+- Deprecate the expansion of shorthand properties (#719)
```

Comment:
Should this instead reference #578, #580, #579, #577, #576, #575, #574, #573, #572, #571, #570, #569, #566, #567, #558, which were the changes on `main`?  And possibly also #714?